gift-certificates-app/dao/src/main/java/com/epam/esm/exception/ResourceHasLinksException.java

```diff
@@ -0,0 +1,31 @@
+package com.epam.esm.exception;
+
```

Comment:
documentation comments are important

Reply:
Public APIs should be documented.  I documented Service. Maybe this is enough?